src/cls/IPM/Main.cls

```diff
@@ -3789,4 +3825,36 @@ ClassMethod Information(ByRef pCommandInfo)
     do ..DisplayModules(.updatedList)
 }
 
+ClassMethod Update(ByRef pCommandInfo)
+{
+    set verbose = $get(pCommandInfo("data","Verbose"))
+    set version = $get(pCommandInfo("parameters","version"))
+    set path = $$$GetModifier(pCommandInfo,"path")
+    set moduleName = $get(pCommandInfo("parameters","module"))
+    set moduleCurrent = ##class(%IPM.Storage.Module).NameOpen(moduleName,.tSC)
```

Comment:
Nit: Use NameExists instead. Also, tSC would be an error here if the module did not exist and so would throw an error before anything further runs